packages/shared/src/components/Attachments/AttachmentItem.tsx

```diff
@@ -107,18 +122,19 @@ export const AttachmentItem = forwardRef<HTMLDivElement, AttachmentItemProps>(
                 {isEditing && (
                     <div
                         data-test-id="attachments-actionbar"
-                        className="tw-flex tw-gap-[2px] tw-invisible group-hover:tw-visible"
+                        className="tw-flex tw-gap-0.5 tw-invisible group-focus:tw-visible group-focus-within:tw-visible group-hover:tw-visible"
                     >
-                        <div {...draggableProps}>
-                            <button
-                                className={joinClassNames([
-                                    'tw-bg-button-background tw-border-button-border hover:tw-bg-button-background-hover active:tw-bg-button-background-pressed tw-group tw-border tw-box-box tw-relative tw-flex tw-items-center tw-justify-center tw-outline-none tw-font-medium tw-rounded tw-h-9 tw-w-9 ',
-                                    isDragging ? 'tw-cursor-grabbing' : 'tw-cursor-grab',
-                                ])}
-                            >
-                                <IconGrabHandle20 />
-                            </button>
-                        </div>
+                        <button
+                            {...focusProps}
+                            {...draggableProps}
+                            className={joinClassNames([
+                                'tw-bg-button-background tw-border-button-border hover:tw-bg-button-background-hover active:tw-bg-button-background-pressed tw-group tw-border tw-box-box tw-relative tw-flex tw-items-center tw-justify-center tw-outline-none tw-font-medium tw-rounded tw-h-9 tw-w-9 ',
+                                isDragging ? 'tw-cursor-grabbing' : 'tw-cursor-grab',
+                                isFocused && FOCUS_STYLE,
+                            ])}
+                        >
+                            <IconGrabHandle20 />
+                        </button>
                         <div className="-tw-mx-3">
                             <Flyout
                                 placement={FlyoutPlacement.Right}
```

Comment:
This flyout trigger is a nested button, which means it has two tab indexes nested inside each other. Use the render function version of the Flyout trigger to avoid this

Reply:
Thanks, fixed!